docs/source/getting-started/installation.rst

```diff
@@ -27,14 +40,14 @@ To install the SDK on Mac or Linux:
 1. Run::
 
      curl -sSL https://get.daml.com/ | sh
-2. If prompted, add ``~/.daml/bin`` to your PATH.
 
-   If you don't know how to do this, try following `these instructions for MacOS <https://hathaway.cc/2008/06/how-to-edit-your-path-environment-variables-on-mac/>`_ or `these instructions for Windows <https://www.java.com/en/download/help/path.xml>`_.
+2. Add ``~/.daml/bin`` to your PATH. You can find the Mac OS and Linux instructions :doc:`here <path-variables>`.
 
 Windows
 =======
 
-We support running the SDK on Windows 10. To install the SDK on Windows, download and run the installer from `github.com/digital-asset/daml/releases/latest <https://github.com/digital-asset/daml/releases/latest>`__.
+1. We support running the SDK on Windows 10. To install the SDK on Windows, download and run the installer from `github.com/digital-asset/daml/releases/latest <https://github.com/digital-asset/daml/releases/latest>`_.
```

Comment:
```suggestion
1. We support running the SDK on Windows 10. To install the SDK on Windows, download and run the installer from https://github.com/digital-asset/daml/releases/latest.
```
You only need the link syntax if you want the text to be something other than the URL.